src/Snt/Capi/Repository/Article/ArticleRepository.php

```diff
@@ -68,8 +64,21 @@ private function fetchArticlesForPublication($publicationId, array $articleIds)
         return isset($articlesRawData['articles']) ? $articlesRawData['articles'] : $articlesRawData;
     }
 
-    private function buildPath($publicationId, array $articleIds)
+    private function buildPath(FindParameters $findParameters)
+    {
+        $articleIds = $findParameters->hasArticleId()
+            ? [$findParameters->getArticleId()]
+            : $findParameters->getArticleIds();
```

Comment:
imho it can be done without ternary operator, like this:
```
$articleIds = array_filter(array_merge([$findParameters->getArticleId()], $findParameters->getArticleIds()))
```

or even you can move this logic to `FindParameters` class, following "tell - don't ask" rule